server/pom.xml

```diff
@@ -744,7 +744,10 @@
       <dependency>
         <groupId>org.zanata</groupId>
         <artifactId>zanata-common-api</artifactId>
+        <!-- release plugin does not play nice with variable substitution
         <version>${zanata.api.compat.version}</version>
+        -->
```

Comment:
We've never had a problem with this one before. I thought the problem was only with recursive substitutions.